src/shared/components/palette/base-field/field-write.component.ts

```diff
@@ -26,11 +26,14 @@ export class FieldWriteComponent extends AbstractFieldWriteComponent implements
   @Input()
   formGroup: FormGroup;
 
+  @Input()
+  caseFields: CaseField[] = [];
```

Comment:
same comment as the one added to field-read.component.ts. `caseFields` should be declared in `AbstractFieldWriteComponent`